src/browser/ui/ReactMount.js

```diff
@@ -451,12 +457,21 @@ var ReactMount = {
    * perform an update on it and only mutate the DOM as necessary to reflect the
    * latest React component.
    *
+   * @param {ReactComponent} parentComponent The conceptual parent of this render tree.
    * @param {ReactElement} nextElement Component element to render.
    * @param {DOMElement} container DOM element to render into.
    * @param {?function} callback function triggered on completion
    * @return {ReactComponent} Component instance rendered in `container`.
    */
-  render: function(nextElement, container, callback) {
+  renderSubtreeIntoContainer: function(parentComponent, nextElement, container, callback) {
+    invariant(
+      parentComponent != null && parentComponent._reactInternalInstance != null,
+      'parentComponent must be a valid React Component'
+    );
+  },
```

Comment:
I'm a little surprised the unit test didn't blow up there.  It seems like travis should have failed this.  Interesting.